adafruit_stmpe610.py

```diff
@@ -199,23 +208,23 @@ def read_data(self):
             self._write_register_byte(_STMPE_INT_STA, 0xFF)
         return (x_loc, y_loc, pressure)
 
-    def _read_byte(self, register):
+    def _read_byte(self, register: int) -> int:
         """Read a byte register value and return it."""
         return self._read_register(register, 1)[0]
 
-    def _read_register(self, register, length):
+    def _read_register(self, register: int, length: int) -> bytearray:
         """Read an arbitrarily long register (specified by length number of
         bytes) and return a bytearray of the retrieved data.
         Subclasses MUST implement this!"""
         raise NotImplementedError
 
-    def _write_register_byte(self, register, value):
+    def _write_register_byte(self, register: int, value: int) -> None:
         """Write a single byte register at the specified register address.
         Subclasses MUST implement this!"""
         raise NotImplementedError
 
     @property
-    def touches(self):
+    def touches(self) -> List[Tuple[int, int, int]]:
```

Comment:
It's actually appending dictionaries to the list with values from the unpacked tuple, so the return is `List[Dict[str, int]]`.